snap/snapcraft.yaml

```diff
@@ -52,7 +54,11 @@ parts:
         rm -rf $CRAFT_PRIME/usr/share/$CRUFT
       done
       find $CRAFT_PRIME/usr/share/doc/ -type f -not -name 'copyright' -delete
-
+   
+  libgconf:
+    plugin: dump
+    source: http://ftp.it.debian.org/debian/pool/main/g/gconf/libgconf-2-4_3.2.6-8_amd64.deb 
```

Comment:
Please use <https://ftp.debian.org/debian/pool/main/g/gconf/libgconf-2-4_3.2.6-8_amd64.deb> as it is backed by a CDN and (theoretically) should be faster to pull.